### PR TITLE
doc(*): create license

### PR DESCRIPTION
**Pull request by calebmer**

### CODE REVIEW

**Comment by calebmer** [comment]
Just going to merge this once tests pass. And look at the commit SHA btw, `0000`! 😊

**Comment by svmnotn** [comment]
uhm... we didn't have a license till now??

**Comment by calebmer** [comment]
Oops, it was in the `Cargo.toml` 😊

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2016 Caleb Meredith
```
Comment (svmnotn) [inline LICENSE:1]: _cough_ _cough_ _cough_

Comment (calebmer) [inline LICENSE:1]: [_cough cough cough_](https://github.com/ardite/ardite-core/graphs/contributors) 😊

Considering the code is like 90% mine I think this is fair.

Comment (svmnotn) [inline LICENSE:1]: for now.

**Pull request merged**